### ISSUE TITLE
Could not create SSL/TLS secure channel

### ISSUE THREAD

**Issue by kendallvision** [tmcdonnell87/OFX-Manager#2]
Thanks for making this available.  When I try to use it for my bank it gets an exception "Could not create SSL/TLS secure channel".  Any insights?  Could it be that the OFX100 out of the box is now outdated and causes the issue?

**Comment by tmcdonnell87** [comment]
Hi! Glad you've found it helpful. TBH I haven't really used the project in 5+ years. 

That said, I would be surprised if the OFX version was causing the error you mention. The SSL/TLS channel should be orthogonal to any content. 

What is the exception message? It may contain some additional clues.

**Comment by kendallvision** [comment]
Thanks for the response despite the age.  Unfortunately, I believe that is all the exception message I am getting.  Any chance you could check if the code works for you still?  I'll dig into the exception and see if there is any additional information.

**Comment by kendallvision** [comment]
Here's the exception message.  {"The request was aborted: Could not create SSL/TLS secure channel."}

**Comment by tmcdonnell87** [comment]
No, sorry, I don't even have VS installed on any of my computers at this point. You're welcome to use it as-is, and I'm happy to provide some guidance based on what I see, but it's not a priority for me to sink a bunch more time into it. 

Very possible there is some SSL versioning issue or out-of-date dependencies, but I can't say for sure.
